### PR TITLE
[#126810153] Gracefully handle errors returned by Tracker

### PR DESCRIPTION
**Pull request by st33b**
Simple PR, with a bit of text made more friendly.

(Spoiler alert, I already did this in `master`, but wanted to get a review before deploying.)

### CODE REVIEW

File: app/models/grouping.rb [hunk 1]
```diff
@@ -229,5 +229,7 @@ def accept_tracker_story
 
     story.current_state = "accepted"
     story.save
+
+    rescue
```
Comment (franckverrot) [inline app/models/grouping.rb:233]: This might bite us in the future, but we have a plan to use current user's tracker API tocker in any case, so... LGTM!

Comment (st33b) [inline app/models/grouping.rb:233]: Yeah, I'm gonna try to implement the "bonus" task in the PT story to use the current user's API key. Figured that should be a separate commit.

Comment (franckverrot) [inline app/models/grouping.rb:233]: Ahah I forgot we added this as a bonus task 😂 

As I forgot, there's a new story in the chores backlog.

**Comment by mpg916** [comment]
LGTM as well. I don't feel the PT integration should stop anyone's workflow and we'll continue to improve as we go so the risk is low.

@st33b Since it's already on master, deploy away!
